lendingbot.py

```diff
@@ -283,8 +292,14 @@ def cancelAndLoanAll():
 		for offer in loanOffers[cur]:
 			onOrderBalances[cur] = onOrderBalances.get(cur, 0) + Decimal(offer['amount'])
 			if dryRun == False:
-				msg = bot.cancelLoanOffer(cur,offer['id'])
-				log.cancelOrders(cur, msg)
+				try:
+					msg = bot.cancelLoanOffer(cur,offer['id'])
+					log.cancelOrders(cur, msg)
+				except Exception as e:
+					log.log("Error canceling loan offer: " + str(e))
+
+				#msg = bot.cancelLoanOffer(cur,offer['id'])
+				#log.cancelOrders(cur, msg)
```

Comment:
Does this need to be changed?

Have you experienced errors with this code? It seems bundled in with the USD feature.